tests/test_quantumGraph.py

```diff
@@ -334,30 +345,56 @@ def testFindCycle(self):
         self.assertFalse(self.qGraph.findCycle())
 
     def testSaveLoad(self):
-        with tempfile.TemporaryFile() as tmpFile:
+        with tempfile.TemporaryFile(suffix='.qgraph') as tmpFile:
             self.qGraph.save(tmpFile)
             tmpFile.seek(0)
             restore = QuantumGraph.load(tmpFile, self.universe)
             self._cleanGraphs(self.qGraph, restore)
             self.assertEqual(self.qGraph, restore)
+            # Load in just one node
+            tmpFile.seek(0)
+            restoreSub = QuantumGraph.load(tmpFile, self.universe, nodes=(0,))
```

Comment:
knowing the implementation, we would always have a node zero, it is just not guaranteed to be the same node. But you are right, it would be better to not depend on that so we can change it at some point in the future.